app/models/users_languages.php

```diff
@@ -160,7 +160,8 @@ public function getUsersForLanguage($lang)
     {
         $result = array(
             'conditions' => array(
-                'language_code' => $lang
+                'language_code' => $lang,
+                'User.group_id != 6'
```

Comment:
Actually I realize there is something missing in the original issue: we want to exclude as well `group_id = 5`.

Functionally speaking, we use `6` for people who are suspended because they did something wrong, and we use `5` for people who ask to have their account deleted but have done nothing wrong (they just don't want to contribute anymore).